yarn.lock

```diff
@@ -8514,6 +8530,25 @@ tslib@^1.8.0, tslib@^1.8.1, tslib@^1.9.0, tslib@^1.9.3:
   resolved "https://registry.npmjs.org/tslib/-/tslib-1.9.3.tgz#d7e4dd79245d85428c4d7e4822a79917954ca286"
   integrity sha512-4krF8scpejhaOgqzBEcGM7yDIEfi0/8+8zDRZhNZZ2kjmHJ4hv3zCbQWxoJGz1iw5U0Jl0nma13xzHXcncMavQ==
 
+tslint@5.14.0:
```

Comment:
@splincode I see that we are running 2 versions of tslint here. Maybe that is causing the travic CI issues?
We could pin it to 5.14, which is required by dtslint.
Please also add the lint task back to the travis build... I would like to try to get this working...

Reply:
Yes!

Reply:
Do you think that is the issue?

Reply:
I need investigation

Reply:
done